inlong-sort/sort-connectors/doris/src/main/java/org/apache/inlong/sort/doris/table/DorisDynamicSchemaOutputFormat.java

```diff
@@ -268,6 +268,13 @@ public void open(int taskNumber, int numTasks) throws IOException {
         if (metricOption != null) {
             metricData = new SinkMetricData(metricOption, getRuntimeContext().getMetricGroup());
         }
+        if (dirtySink != null) {
+            try {
+                dirtySink.open(new Configuration());
+            } catch (Exception e) {
+                throw new IOException(e);
```

Comment:
why throw IOException?

Reply:
Initialization failed, not throwing an exception will cause problems in subsequent processing.